src/Backends/Cubics/GeneralizedCubic.h

```diff
@@ -119,13 +119,13 @@ class AbstractCubic
     double get_Delta_2(){ return Delta_2; }
     /// Read-only accessor for value of R_u (universal gas constant)
     double get_R_u(){ return R_u; }
-    /// Set the three Mathias-Copeman constants in one shot for a pure fluid
-    void set_C_MC(double c1, double c2, double c3){
-        alpha[0].reset(new MathiasCopemanAlphaFunction(a0_ii(0),c1,c2,c3,T_r/Tc[0]));
+    /// Set the three Mathias-Copeman constants in one shot for the component i of a mixturee
```

Comment:
mixturee -> mixture

This will break my code, but I am fine with that since I doubt anyone else is working so deep inside the internals of CoolProp.